src/Ops.Controllers/Areas/Admin/FeatureController.cs

```diff
@@ -163,6 +164,17 @@ public async Task<IActionResult> DeleteFeature(Feature feature)
         [SaveModelState]
         public async Task<IActionResult> EditFeature(Feature feature)
         {
+            if (feature.NeedsPopup && string.IsNullOrEmpty(feature.Stub))
+            {
+                ModelState.AddModelError("Feature.Stub", "A 'Stub' is required for a popup.");
+                ShowAlertDanger($"Invalid paramaters");
```

Comment:
This doesn't seem like a good error to be showing the end user. Also, strings do not need the interpolation character (`$""`) if there are no `{variables}` in them.